meta-client/src/main/java/org/apache/helix/metaclient/factories/MetaClientFactory.java

```diff
@@ -49,4 +50,20 @@ public MetaClientInterface getMetaClient(MetaClientConfig config) {
     }
     return null;
   }
+
+  public MetaClientCacheInterface getMetaClientCache(MetaClientConfig config, MetaClientCacheConfig cacheConfig) {
+    if (config == null) {
+      throw new IllegalArgumentException("MetaClientConfig cannot be null.");
+    }
+    if (MetaClientConfig.StoreType.ZOOKEEPER.equals(config.getStoreType())) {
+      ZkMetaClientConfig zkMetaClientConfig = new ZkMetaClientConfig.ZkMetaClientConfigBuilder().
+              setConnectionAddress(config.getConnectionAddress())
+              .setMetaClientReconnectPolicy(config.getMetaClientReconnectPolicy())
+              .setConnectionInitTimeoutInMillis(config.getConnectionInitTimeoutInMillis())
+              .setSessionTimeoutInMillis(config.getSessionTimeoutInMillis())
+              .build();
+      return new ZkMetaClientFactory().getMetaClientCache(zkMetaClientConfig, cacheConfig);
```

Comment:
Forgot to mention it in prev comment.  line 60-65 can be merged with line 44-49. Please address it in following PR.